spec/features/workflow/submit_for_2i_spec.rb

```diff
@@ -45,6 +49,7 @@ def and_i_see_a_link_to_the_edition
 
   def when_i_edit_the_edition
     stub_any_publishing_api_put_content
+    visit document_path(@edition.document)
```

Comment:
Should this be `click_on "Document summary"`

Reply:
Sure